dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/OpenAIFunction.cs

```diff
@@ -77,7 +77,7 @@ public class OpenAIFunction
     /// <summary>
     /// Separator between the plugin name and the function name
     /// </summary>
-    public const string NameSeparator = "-";
+    public const string NameSeparator = "_";
```

Comment:
Glad we made this change as using `-` made it look like a matchs operation. Was there a use case that caused the old separator to change?

Reply:
@alliscode made this change - not sure if it was in response to a particular use case or not.